scanner/cmd/updater/main.go

```diff
@@ -30,27 +33,42 @@ func tryExport(outputDir string) error {
 }
 
 func main() {
+	versionFlag := flag.Bool("version", false, "Print version")
 	outputDir := flag.String("output-dir", "", "Output directory")
+	dbConn := flag.String("db-conn", "", "Postgres connection string")
+	vulnsURL := flag.String("vulns-url", "", "URL to the vulnerabilities bundle")
 	flag.Parse()
 
-	if *outputDir == "" {
-		log.Fatal("Missing argument for the output directory.")
+	if *versionFlag {
+		fmt.Println(version.Version)
```

Comment:
log vs print?

Reply:
Log will add timestamps and whatnot -- I want this to only print the version so I can potentially use it in scripts.